R/plotDataSetList.R

```diff
@@ -75,7 +75,7 @@ grad_functions <- c(
 #' Plot.RT.Single_Func(subset(dsl, funcId == 1))
 Plot.RT.Single_Func <- function(dsList, Fstart = NULL, Fstop = NULL,
                                show.ERT = T, show.CI = F, show.mean = F,
-                               show.median = F, backend = 'plotly',
+                               show.median = F, backend = NULL,
```

Comment:
Because I check if they are NULL and then set them to the default value. I don't think using getOption(...) in the function header would work, but I'm not sure